OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -80,8 +81,7 @@ public EditorActorPreview(WorldRenderer worldRenderer, string id, ActorReference
 				Footprint = ios.OccupiedCells(Info, location, subCell);
 			else
 			{
```

Comment:
Can remove these braces now, or possibly replace this whole thing with
```csharp
Footprint = ios?.OccupiedCells(Info, location, subCell) ?? new Dictionary<CPos, SubCell>() { { location, SubCell.FullCell } };
```
(but maybe a little long?)